_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+title: Analyze Jaeger trace data 
+parent: Trace analytics
+nav_order: 55
+---
+
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
+
+The Trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace analytics built-in analysis capabilities. This provides support for OpenTelemetry (OTEL) formatted trace data.
+
+When you perform trace analytics, you can select from two data sources:
+
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper.
+- **Jaeger** – Trace data stored within OpenSearch as its backend.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace Analytics to analyze the error rates and latencies You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
+
+When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
```

Comment:
What kind of configuration? Should we give user an example of what they may have to configure?